added human prompting description (#223)
RapidataAI/rapidata-python-sdk: docs/human_prompting.md

# Effective Instruction Design for Rapidata Tasks

> When creating tasks for human annotators using the Rapidata API, the way you phrase your instructions can significantly impact the quality and consistency of the responses you receive. This guide provides best practices for designing effective instructions for your Rapidata tasks.

Reviewer: ```suggestion
When creating tasks for human annotators using the Rapidata API, phrasing your instructions well can significantly improve quality and consistency of the responses you receive. This guide provides best practices for designing effective instructions for your Rapidata tasks.
```

### Use Positive Framing
Frame questions in the positive rather than negative. Positive questions are easier to process quickly.

**Better:**
```
"Which image looks more realistic?"
```

**Avoid:**
```
"Which image looks less AI-generated?"
```

### Limit Decision Criteria
Don't overload annotators with multiple criteria in a single question.

**Better:**
```
"What animal is in the image? - rabbit/dog/cat/other"
```

**Avoid:**
```
"Does this image contain a rabbit, a dog, or a cat? - yes/no"
```

### Use Clear Response Options
Provide distinct, non-overlapping response options.

**Better:**
```
"Rate the image quality: poor/acceptable/excellent"
```

**Avoid:**
```
"Rate the image quality: bad/not good/fine/good/great"
```

## Task Sequence Considerations

When using multiple selections in a session, consider the cognitive load and time constraints:

```python
from rapidata import LabelingSelection, ValidationSelection

# Keep the total number of tasks manageable for the 25-second window
selections=[
    ValidationSelection("67cafc95bc71604b08d8aa62", 1),  # Start with one validation task (id is the validation set id)

>     LabelingSelection(1)  # Follow with one labeling task

Reviewer: I think the standard should still be 2 for most things

```suggestion
    LabelingSelection(2)  # Follow with one labeling task
```

]
```

## Example Implementation

When creating a Rapidata order, implement these principles as follows:

```python
order = rapi.order.create_compare_order(
    name="Image Coherence Comparison",
    instruction="Which images has more glitches and is more likely to be AI generated?",  # Clear, positive framing
    datapoints=[["https://assets.rapidata.ai/flux-1.1-pro/33_2.jpg", 
    "https://assets.rapidata.ai/stable-diffusion-3/33_0.jpg"]],
    selections=selections  # Include the above-defined selections
)
```

## Common Task Types and Recommended Instructions

### Image Comparison Tasks

```python
# Comparing image preference
instruction="Which image do you prefer?"

# Comparing prompt adherence
instruction="Which image matches the description better?"

# Comparing image coherence
instruction="Which images has more glitches and is more likely to be AI generated?"

# Comparing two texts

> instruction="Which of these sentences make more sense?"

Reviewer: ```suggestion
instruction="Which of these sentences makes more sense?"
```
